src/demos/linking-it-all/app.js

```diff
@@ -161,43 +134,46 @@ export default class App extends Component {
     });
   }
 
+  _updateLayerSettings(settings) {
+    this.setState({settings});
+  }
+
   render() {
-    const {viewport, hoveredObject, 
-      points, settings, status, x, y,
-      highlightedHour, selectedHour
-    } = this.state;
     return (
       <div>
-        {hoveredObject &&
-          <div style={{...tooltipStyle, left: x, top: y}}>
-            <div>{hoveredObject.id}</div>
+        {this.state.hoveredObject &&
+          <div style={{
+            ...tooltipStyle,
+            left: this.state.x,
+            top: this.state.y
+          }}>
+            <div>{this.state.hoveredObject.id}</div>
           </div>}
-        <LayerControls
-          settings={settings}
-          propTypes={LAYER_CONTROLS}
-          onChange={this.updateLayerSettings.bind(this)}
-        />
+        {this.props.noControls ? null : <LayerControls
```

Comment:
I would prefer having a default prop set to true and passing false in the `Hero`.

```js
{this.props.showControls && (
  <LayerControls ... />
)}
```